### ISSUE TITLE
Linking issue in node 10 and 11

### ISSUE THREAD

**Issue by bkimminich** [marudor/libxmljs2#3]
When used as a drop in replacement for `libxmljs` in node 10 and 11 this happens:

```
Error: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `CXXABI_1.3.9' not found (required by /home/travis/build/bkimminich/juice-shop/node_modules/libxmljs2/build/Release/xmljs.node)
```

https://travis-ci.org/bkimminich/juice-shop/builds/535071336?utm_medium=notification&utm_source=email

**Comment by marudor** [comment]
Please try again with 20.0.2
I'm using the stretch docker image to build it now. That should support your glibc version.

**Comment by marudor** [comment]
Okay - nvm it does not. Grr. Gonna fix it later today.

**Comment by marudor** [comment]
Fixed with 20.0.3 (I used your branch to test it)
